ci-operator/step-registry/vsphere/provision/bastionhost/vsphere-provision-bastionhost-commands.sh

```diff
@@ -0,0 +1,151 @@
+#!/bin/bash
+
+set -o nounset
+set -o errexit
+set -o pipefail
+
+trap 'CHILDREN=$(jobs -p); if test -n "${CHILDREN}"; then kill ${CHILDREN} && wait; fi' TERM
+
+CLUSTER_NAME="${NAMESPACE}-${JOB_NAME_HASH}"
+bastion_name="${CLUSTER_NAME}-bastion"
+bastion_ignition_file="${SHARED_DIR}/${CLUSTER_NAME}-bastion.ign"
+if [[ ! -f "${bastion_ignition_file}" ]]; then
+  echo "'${bastion_ignition_file}' not found, abort." && exit 1
+fi
+bastion_ignition_base64=$(base64 -w0 < "${bastion_ignition_file}")
+
+echo "$(date -u --rfc-3339=seconds) - Configuring govc exports..."
+# shellcheck source=/dev/null
+source "${SHARED_DIR}/govc.sh"
+
+#Get available ova template
+echo "$(date -u --rfc-3339=seconds) - Get avaiable ova template..."
```

Comment:
For bastion host, we do not have to update the image frequently, we just need to ensure the bastion host can be launched and works well. So similar to gcp and azure, we can introduce a ref env for the ova disk, and give a default value there.

Reply:
I read this part on gcp and azure, a little difference is that ova image uploaded in Prow CI maybe removed by pruner script or other script periodically, so before using it, need to check and uploaded again if not exists.
If introduce the env for ova, we may need to maintain that template manually?

Reply:
We can introduce a ref env var, like BASTION_OVA_URI, and set it to a well-tested ova disk, e.g: https://rhcos.mirror.openshift.com/art/storage/releases/rhcos-4.10/410.84.202207061638-0/x86_64/rhcos-410.84.202207061638-0-vmware.x86_64.ova, not have to always get the latest OVA disk url from `openshift-install coreos print-stream-json`. 

For the pruner issue, you can do the same logic, before using it,  need to check and uploaded again if not exists.

Reply:
> need to check and uploaded again if not exists.

I'm not sure if any potential issue here, will it hit errors while multi-jobs upload the image at the same time?

Reply:
vsphere upi installation in Prow CI have one step to check ova and uploaded if needed, seems not detect conflict issue so far.
https://github.com/openshift/release/blob/master/ci-operator/step-registry/upi/conf/vsphere/ova/upi-conf-vsphere-ova-commands.sh#L36-L44